.github/actions/setup_env/action.yml

```diff
@@ -30,3 +30,6 @@ runs:
           environment-name: tardis
           cache-environment: true
           cache-downloads: true
+      
+      - run: export PANEL_DOCS_BUILD=1
```

Comment:
What does this do?

Reply:
this was to determine which environment the code was being run in, for live updates the panel extension requires the ipywidget comms while the same does not work in documentation build. this was used to check and apply the appropriate extension comms module for panel. but I changed this now and moved it to conf.py without environment variables so it is much cleaner.